pyproject.toml

```diff
@@ -22,15 +22,15 @@ readme="README.rst"
 python = "^3.7.1"
 
 [tool.poetry.dev-dependencies]
-black = ">=21.5b1,<24"
-ipython = ">=7.0,<9.0"
-mypy = ">=0.901,<1"
-pytest = ">=6.2.4,<8"
-pytest-cov = ">=2.12.1,<4.1"
-pytest-mock = ">=3.6.1,<3.11"
+black = "^21.5b1"
```

Comment:
doesn't this top it at v22?

Reply:
had a bit of a fight, but we are now I think in a better place